### ISSUE TITLE
Multiple errors on startup.

### ISSUE THREAD

**Issue by m1ksu** [JBakamovic/yavide#77]
![screenshot_2017-10-14_01-25-29](https://user-images.githubusercontent.com/22960424/31568737-c0f8fe48-b07e-11e7-847e-2fb3884a5f35.png)
Why does this happen and how do I fix this?

**Comment by JBakamovic** [comment]
Seems like you don't have python support compiled in for your version of vim.

**Comment by nolanholden** [comment]
It appears that for Ubuntu 17.04 Zesty, `apt install vim-gtk` installs vim _without_ python support, only with python3 support.

Even with python2 support, this issue persists:

I tried to solve this problem by first uninstalling vim (**DANGER: take note of any files, such as your vimrc, which may be lost.**). To be sure everything is uninstalled:
```shell
sudo apt-get remove *vim*
```
_Caution: this may match some non-vim packages._

And then, compiling ViM from source:
```shell
cd /tmp && git clone https://github.com/vim/vim.git && cd vim
./configure --enable-pythoninterp --prefix=/usr
make && sudo make install
```
^[credits](https://vi.stackexchange.com/questions/11526/how-to-enable-python-feature-in-vim)

Now, when I run `vim --version`, the list of features shows:
```
...
+python    # <--- this is what we were lacking
-python3
...
```

**Comment by JBakamovic** [comment]
Yes, it is crucial that `gvim` is compiled with `python2` support otherwise it will not work (most source code in `yavide` is `python2`). Some distros decide to distribute and compile `gvim` with `python3` support only, some others compile it with `python2` only and the rest compile it with both `python2` & `python3` support. This is where the things start tearing apart.

What people can do to fix this problem is to install required dependencies in order to get `gvim` with `python2` support. Hopefully someone will make a pull-request as well to fix the installation script as well.

**Comment by linbinchen** [comment]
I am on Mint 18.3 which should based on Ubuntu 16.04. I encountered the same issue on startup. My solution is: After installation, I change the yavide.desktop to use vim.gnome-py2 -g to launch yavide. Because I am using python 3 as my default python. So, before installation, I also change the "PIP_INSTALL_CMD" in install.sh to "pip2 install".

**Comment by JBakamovic** [comment]
Thanks for reporting the solution back.
